h/static/scripts/annotation-ui.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
+var immutable = require('seamless-immutable');
+var redux = require('redux');
+
 function value(selection) {
```

Comment:
Just in case there was confusion here, I was talking about the function on line 6, currently called `value`.